src/audio/mux/mux_generic.c

```diff
@@ -18,6 +18,7 @@
 #include <stddef.h>
```

Comment:
In this commit I presume you just move things around so you can guard them with "#if CONFIG_..." no? (the diff is thus misleading)

Reply:
ok, I split moving grouping function to separate commit